--- origsrc/avant-window-navigator-0.2.6/awn-applet-activation/Makefile.am	2008-02-22 10:10:17.000000000 -0600
+++ src/avant-window-navigator-0.2.6/awn-applet-activation/Makefile.am	2008-10-28 21:35:41.176908700 -0500
@@ -1,5 +1,6 @@
 NULL =
 INCLUDES =		\
+	-I$(top_srcdir)	\
 	$(AWN_CFLAGS)	\
 	$(NULL)
 
--- origsrc/avant-window-navigator-0.2.6/data/Makefile.am	2008-02-22 10:10:17.000000000 -0600
+++ src/avant-window-navigator-0.2.6/data/Makefile.am	2008-10-28 21:39:59.179323300 -0500
@@ -31,7 +31,7 @@
 endif
 
 install-exec-local:
-	$(INSTALL) -m 755 awn-schema-to-gconf $(DESTDIR)$(bindir)/awn-schema-to-gconf
+	$(INSTALL) -m 755 $(srcdir)/awn-schema-to-gconf $(DESTDIR)$(bindir)/awn-schema-to-gconf
 
 uninstall-local:
 	rm -f $(DESTDIR)$(bindir)/awn-schema-to-gconf
--- origsrc/avant-window-navigator-0.2.6/src/Makefile.am	2008-02-22 10:10:18.000000000 -0600
+++ src/avant-window-navigator-0.2.6/src/Makefile.am	2008-10-28 21:31:03.663302900 -0500
@@ -4,6 +4,7 @@
 	$(DOCK_CFLAGS)			\
 	-DLIBDIR=\""$(libdir)"\"	\
 	-I$(top_srcdir)/libawn		\
+	-I$(top_srcdir)		\
 	$(NULL)
 
 bin_PROGRAMS = avant-window-navigator
